serverlessworkflow/modules/ROOT/pages/getting-started/preparing-environment.adoc

```diff
@@ -0,0 +1,88 @@
+. Install https://knative.dev/docs/install/quickstart-install/[Knative using quickstart]. This will also setup Knative Serving and Eventing for you and the cluster should be running.
+. xref:cloud/operator/install-serverless-operator.adoc[]
+. Install xref:testing-and-troubleshooting/kn-plugin-workflow-overview.adoc[Knative Workflow CLI].
+. Install https://code.visualstudio.com/[Visual Studio Code] with https://marketplace.visualstudio.com/items?itemName=kie-group.swf-vscode-extension[our extension] that simplifies development of workflows by provifing visual aids and auto-complete features.
+
+[[proc-starting-cluster-fo-local-development]]
+== Starting the cluster for local development
+
+If you have used https://knative.dev/docs/install/quickstart-install/[Knative using quickstart] guide, your selected cluster should be running and properly configured to work with our guides.
+
+Please note, that if the knative quickstart procedure is not used, you need to install Knative Serving and Eventing manually. See <<proc-additional-options-for-local-environment>>.
+
++
+.To startup the selected cluster without quickstart, use the following command:
+[tabs]
+====
+Minikube::
++
+--
+.Configure and startup minikube
+[source,shell]
+----
+# Set a driver and container runtime
+minikube config set driver docker/podman
+minikube config set container-runtime docker/podman
+
+# Set cpu and memory
+minikube config set cpus 4
+minikube config set memory 4096
+
+# Start the cluster
+minikube start --cpus 4 --memory 4096 --addons registry --addons metrics-server --insecure-registry "10.0.0.0/24" --insecure-registry "localhost:5000"
+
+# Set the active profile
+minikube profile minikube
+----
+--
+Kind::
++
+--
+.Start a kind cluster
+[source,shell]
+----
+kind create cluster
+----
+--
+====
+
+[[proc-advanced-local-environment-setup]]
+== Advanced local environment setup
+
+If you are interested in our Java and Quarkus development path, consider completing this procedure in addition to the
+<<proc-minimal-local-environment-setup>>. By completing these steps you are able to start the development of applications on your local machine using our xref:use-cases/advanced-developer-use-cases/index.adoc[advanced developer guides].
+
+.Procedure
+. Install https://openjdk.org/[OpenJDK] {java_min_version} and cofigure `JAVA_HOME` appropriately by adding it to the `PATH`.
+. Install https://maven.apache.org/index.html[Apache Maven] {maven_min_version}.
+. Install https://quarkus.io/guides/cli-tooling[Quarkus CLI] corresponding to currently supported version by {product_name}. Currently it is {quarkus_version}.
+
+[[proc-additional-options-for-local-environment]]
+== Additional options for local environment setup
+
+* Install https://www.graalvm.org/[GraalVM] {graalvm_min_version}. This will allow you to create https://www.graalvm.org/22.0/reference-manual/native-image/[native image] of your {product_name} application.
```

Comment:
Maybe we can remark somehow that this is an "Optional" step, that you need only in that case.
Why? well, I believe that this is a kind of advanced use case that people normally won't use.
For instance, if you want to play with Dev/Preview (former Prod) profiles, mode, you'll never need it.
So we reduce in one step environment preparation.

Reply:
:+1:  this part should have optionals - will weave that it